docs/reporting.rst

```diff
@@ -725,17 +723,76 @@ lead organisation" line.
+`Active Episodes <https://docs.pmhc-mds.com/projects/data-specification/en/latest/data-model-and-specifications.html#active-episode>`_,
 and the number of episodes delivered at multiple organisations is what is reported in the "Active Episodes" column.
 
 The "Service Contacts" column counts all the non no-show contacts in the
-reporting period that are associated with the `Active Episodes <https://docs.pmhc-mds.com/projects/data-specification/en/v2/data-model-and-specifications.html#active-episode>`_.
-Similarly the `Active Clients <https://docs.pmhc-mds.com/projects/data-specification/en/v2/data-model-and-specifications.html#active-client>`_
+reporting period that are associated with the `Active Episodes <https://docs.pmhc-mds.com/projects/data-specification/en/latest/data-model-and-specifications.html#active-episode>`_.
+Similarly the `Active Clients <https://docs.pmhc-mds.com/projects/data-specification/en/latest/data-model-and-specifications.html#active-client>`_
 is the enumeration of all clients for whom ALL episodes
 active during the reporting period were delivered at multiple organisations.
 A single episode during the reporting period delivered at only one
 organisation excludes client from this count.
 
+.. _category-a13:
+
+A13 — Data quality report — Universal Aftercare episodes with no linked UA episode record
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+The A13 provides an overview of the number of episodes that have the 
+Universal Aftercare Program Type but no linked UA Episode record.
+
+Key specifications:
+
+* `Active Episodes <https://docs.pmhc-mds.com/projects/data-specification/en/latest/data-model-and-specifications.html#active-episode>`_ with Program Type 'Universal Aftercare' that do not have an associated UA Episode record
+
+.. _category-a14:
+
+A14 — Data quality report - Missing UA Episode data
++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+The A14 is a summary Universal Aftercare (UA) format report that provides a simple
+listing of missing/invalid data rates for relevant UA Episode data elements. It
+is analogous to the `standard A4 report <https://docs.pmhc-mds.com/projects/user-documentation/en/latest/reporting.html#a4-data-quality-report-missing-and-invalid-episode-data>`_
+that deals with the standard PMHC Episode elements with missing data.
+
+Key specifications:
+
+* Data in this report is associated with an Active UA Episode; however, for this 
+  report there is slightly tighter criteria regarding what it means to be active. 
+  Rather than the usual one or more 
+  `attended service contact <https://docs.pmhc-mds.com/projects/data-specification/en/latest/data-model-and-specifications.html#attended-service-contact>`_ 
+  during the reporting period, for the A14, there must be at least TWO such contacts (i.e. non-no shows) during the reporting period. 
```

Comment:
Oh, no. I just checked… Looks like I screwed up here and removed the ‘at least TWO’ in the code.